src/Uno.UI/UI/Xaml/Controls/Flyout/Flyout.cs

```diff
@@ -99,6 +98,8 @@ private void OnPopupOpened(object sender, object e)
 			}
 		}
 
+		private void OnChildSizeChanged(object sender, SizeChangedEventArgs args) => SetPopupPositionPartial(Target);
```

Comment:
is this new method used?

Reply:
ah oui j'avais oublier de l'enlever quand j'ai reverter le code a serialdispose